litmus-portal/frontend/src/components/Header/NotificationDropdown.tsx

```diff
@@ -14,33 +14,39 @@ import {
   Badge,
 } from '@material-ui/core';
 import NotificationsOutlinedIcon from '@material-ui/icons/NotificationsOutlined';
-import NotificationListItem from '../NotificationListItem';
+import NotificationListItem from './NotificationListItem';
 import useStyles from './styles';
+import { Message, NotificationsCallBackType } from '../../models/header';
 
-interface NotifierProps {
-  messages: any;
-  count: any;
+interface NotificationDropdownProps {
+  messages: Message[];
+  count: string;
+  CallbackToHeaderOnDeleteNotification: NotificationsCallBackType;
 }
 
-function NotificationsPopperButton(props: NotifierProps) {
+function NotificationsDropdown(props: NotificationDropdownProps) {
```

Comment:
Since we are following the standards of writing a functional component with React.FC, you can rewrite this as
```
const NotificationsDropdown:React.FC<NotificationDropdownProps> = ({ 
messages, 
count, 
CallbackToHeaderOnDeleteNotification }) => {
```